CMakeLists.txt

```diff
@@ -2,15 +2,24 @@ PROJECT(qmcpack)
 
 
 #####################################################
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.10)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.6.0)
 IF(COMMAND cmake_policy)
   cmake_policy(SET CMP0003 NEW)
-  IF(CMAKE_VERSION VERSION_GREATER 3.1.3 AND NOT $ENV{CRAYPE_VERSION} MATCHES ".")
+  IF (COMPILING_ON_BGQ)
```

Comment:
Is it because of HDF5_USE_STATIC_LIBRARIES?

Reply:
Yes.

Reply:
Is it possible to put an if around HDF5_USE_STATIC_LIBRARIES=off instead of changing the policy.

Reply:
The HDF5_USE_STATIC_LIBRARIES flag seems to have no effect on BG/Q - the list of libraries is the same.
HDF libraries =
/soft/libraries/hdf5/current/cnk-gcc/current/lib/libhdf5.a;/soft/libraries/alcf/current/gcc/ZLIB/lib/libz.a;/usr/lib64/libdl.so;/usr/lib64/libm.so

Reply:
I see. I think CMake is doing right because HDF5_USE_STATIC_LIBRARIES only means HDF5  is static but the overall linking is still dynamic by default.